tfhe/src/shortint/client_key/mod.rs

```diff
@@ -72,6 +72,61 @@ impl ClientKey {
         ShortintEngine::with_thread_local_mut(|engine| engine.encrypt(self, message).unwrap())
     }
 
+    /// A trivial encryption uses a zero mask and no noise.
+    ///
+    /// It is absolutely not secure, as the body contains a direct copy of the plaintext.
+    /// However, it is useful for some FHE algorithms taking public information as input.
+    ///
+    ///  By definition a trivial encryption can be decrypted by any size-compatible [`ClientKey`].
+    ///
+    /// Trivially encrypt an input message in a [`Ciphertext`].
+    ///
+    /// # Example
+    ///
+    /// ```rust
+    /// use tfhe::shortint::parameters::PARAM_MESSAGE_2_CARRY_2;
+    /// use tfhe::shortint::ClientKey;
+    ///
+    /// let cks = ClientKey::new(PARAM_MESSAGE_2_CARRY_2);
+    ///
+    /// // Insecure trivial encryption of one message that is within the encrypted message modulus:
+    /// let msg = 3;
+    /// let ct = cks.trivial_encrypt(msg);
+    ///
+    /// let dec = cks.decrypt(&ct);
+    /// assert_eq!(msg, dec);
+    ///
+    /// // Unrelated ClientKey, to show the encryption is not secure
+    /// let other_cks = ClientKey::new(PARAM_MESSAGE_2_CARRY_2);
+    ///
+    /// let dec = other_cks.decrypt(&ct);
+    /// assert_eq!(msg, dec);
+    ///
+    /// // Insecure trivial encryption of one message that is outside the encrypted message modulus:
+    /// let msg = 5;
+    /// let ct = cks.trivial_encrypt(msg);
+    ///
+    /// let dec = cks.decrypt(&ct);
+    /// let modulus = cks.parameters.message_modulus.0 as u64;
+    /// assert_eq!(msg % modulus, dec);
+    ///
+    /// // Decryption with unrelated key, showing the encryption is not secure
+    /// let dec = other_cks.decrypt(&ct);
+    /// let modulus = other_cks.parameters.message_modulus.0 as u64;
+    /// assert_eq!(msg % modulus, dec);
+    /// ```
+    pub fn trivial_encrypt(&self, message: u64) -> Ciphertext {
```

Comment:
To me, its the server who is most likely going to need the ability to do trivial encryption.
Like initializing a variable to a trivial 0 before soing a sum

Reply:
they are server side, changing this PR to avoid footguns on the client side